core/frontend/src/tile/IndexedDBCache.ts

```diff
@@ -0,0 +1,185 @@
+
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @beta */
```

Comment:
Why do we want these APIs to be part of the package's public API? They seem like implementation details.